src/Routes/ImageManager/Images.js

```diff
@@ -40,18 +47,29 @@ const Images = () => {
   const [hasModalSubmitted, setHasModalSubmitted] = useState(false);
 
   const openCreateWizard = () => {
-    history.push({
+    const a = {
       pathname,
       search: stateToUrlSearch('create_image=true', true, search),
-    });
+    };
+    if (navigateProp) {
+      navigate({ ...a, replace: true });
+    } else {
+      history.push({ a });
```

Comment:
We need to remove the curly braces:
```suggestion
      history.push(a);
```